### PR TITLE
Fix and clean up scenario app for Android

### PR DESCRIPTION
**Pull request by blasten**
## Description

Creates a style that hides the makes the activity full screen, and hides the status bar. This way, there's no need to crop the screenshot used for comparisons. 

Also, cleans up the tests a bit:
  * Remove `builder.pushOffset(0,0)` as that doesn't do anything.

## Related Issues

https://github.com/flutter/flutter/issues/70633

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [contributor guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [C++, Objective-C, Java style guides] for the engine.
- [x] I read the [tree hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation.
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Reviewer Checklist

- [ ] I have submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

## Breaking Change

Did any tests fail when you ran them? Please read [handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[contributor guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[tree hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[engine presubmit flakes form]: https://forms.gle/Wc1VyFRYJjQTH6w5A

### CODE REVIEW

File: testing/scenario_app/lib/src/platform_view.dart [hunk 1]
```diff
@@ -357,13 +341,13 @@ class MultiPlatformViewBackgroundForegroundScenario extends Scenario with _BaseP
   void _firstFrame() {
     final SceneBuilder builder = SceneBuilder();
 
-    builder.pushOffset(0, 0);
-
-    builder.pushOffset(0, 600);
+    builder.pushOffset(50, 600);
```
Comment (cyanglaz) [inline testing/scenario_app/lib/src/platform_view.dart:344]: Why do we need to add 50 on x?

Comment (blasten) [reply testing/scenario_app/lib/src/platform_view.dart:344]: The screenshot will explain it better, see before/after.

Comment (blasten) [reply testing/scenario_app/lib/src/platform_view.dart:344]: https://github.com/flutter/engine/pull/22540/files?short_path=d68b0a5#diff-d68b0a59cd0063d730bf092fc5279ee749dd4e9348710757339c80d3a68f7923

**Review by cyanglaz — APPROVED** [review]
LGTM

**Pull request merged**